contracts/src/libraries/EigenDACertVerificationV2Lib.sol

```diff
@@ -237,12 +215,6 @@ library EigenDACertVerificationV2Lib {
         pure
         returns (StatusCode err, bytes memory errParams)
     {
-        if (securityThresholds.confirmationThreshold <= securityThresholds.adversaryThreshold) {
-            return (
-                StatusCode.INVALID_THRESHOLD_PERCENTAGES,
-                abi.encode(securityThresholds.confirmationThreshold, securityThresholds.adversaryThreshold)
-            );
-        }
         uint256 gamma = securityThresholds.confirmationThreshold - securityThresholds.adversaryThreshold;
         uint256 n = (10000 - ((1_000_000 / gamma) / uint256(blobParams.codingRate))) * uint256(blobParams.numChunks);
         uint256 minRequired = blobParams.maxNumOperators * 10000;
```

Comment:
Most likely unrelated to this PR, but I'm a bit puzzled as to this checkSecurityParams function overall, since it related VersioninBlobParams (which are eigenda set) and securityThresholds (which are rollup set). So certs could randomly start not verifying because of a mismatch between the options chosen by rollup and eigenda independently it seems..